arrays&objects/studentDirectory.js

```diff
@@ -1,3 +1,6 @@
+//Information for each student is stored in an object
+// From those objects, functions pull information about age,email addresses and programming languages.
+// stores them in arrays and outputs them  
```

Comment:
@NosiphoManyase Your code looks fantasic! I see your below changes are already in the `main` branch so I can't comment on each line individually, but I'll leave some comments here :) 

**Lines 7 - 11**
Data looks great. Another solution could have been to make another variable called `students` that stored all the students within it. ie.
```
const students = [student1, student2, student3, student4, student5]
```

** Function names ** 
In JS it's usually convention to use camelcase for naming. ie `getYoungestStudent`. Underscores are a Python convention.

**Line 33**
Nice use of the `search` function 👏

Reply:
@ni3ol , Thank  you for the feedback.

Lines 7- 11
I don't know why I didn't think of that. I just didn't create a variable to store students because I didn't need it , is it then for best practice? or...

**Function names **
I knew I should've used camelCase, lol. But I took the shortcut and copied and pasted. I'll definitely take note to use strictly camelCase moving foward

Reply:
@NosiphoManyase you can merge this Pull request when you're ready :)